src/Aspire.Cli/Resources/InteractionServiceStrings.resx

```diff
@@ -200,4 +200,12 @@
   <data name="WaitingForDebuggerToAttachToAppHost" xml:space="preserve">
     <value>Waiting for debugger to attach to app host process</value>
   </data>
+  <data name="NewDashboardVersionAvailable" xml:space="preserve">
```

Comment:
this resource name seems wrong. NewCLIVersionAvailable? And below, MoreInfoNewCLIVersion?